Update README.md (#6)
NishanthMuruganantham/ndtv-api: README.md

<p>This API will scrap the news content present in the NDTV website will provide the data in a JSON format.</p>

> URL : https://ndtvnews-api.vercel.com

Reviewer: ```suggestion
URL : https://ndtvapi.vercel.app/
```

<p>Please find the instructions for using the respective endpoints.</p>

> <h3>For https://ndtvnews-api.vercel.com/general :-</h3>

Reviewer: ```suggestion
<h3>For https://ndtvapi.vercel.app/general :-</h3>
```

Key | Name |Example
---------|----------|---------

>  category | news category you wish to get | https://ndtvnews-api.vercel.com/general?category=values(latest,india,world)

Reviewer: ```suggestion
 category | news category you wish to get | https://ndtvapi.vercel.app/general?category=values(latest,india,world)
```
